packages/storybook/src/templates/mijn-omgeving-home/index.css

```
@@ -249,29 +285,20 @@
 }
 
 .todo-page-footer__spotlight {
-  --ams-page-footer-menu-link-color: var(--voorbeeld-color-white);
-  --ams-page-footer-menu-hover-color: var(--voorbeeld-color-gray-100);
-  --utrecht-heading-2-color: var(--voorbeeld-color-white);
-  --utrecht-heading-2-padding-start: 2em;
-  --utrecht-paragraph-color: var(--voorbeeld-color-white);
-  --utrecht-link-color: var(--voorbeeld-color-white);
-  --utrecht-link-hover-color: var(--voorbeeld-color-gray-100);
+  --ams-page-footer-menu-link-color: var(--todo-page-footer-menu-link-color);
+  --ams-page-footer-menu-hover-color: var(--todo-page-footer-menu-hover-color);
+  --utrecht-heading-4-color: var(--todo-page-footer-heading-4-color);
+  --utrecht-heading-4-margin-block-end: var(--voorbeeld-space-block-beetle);
+  --utrecht-paragraph-color: var(--todo-page-footer-paragraph-color);
+  --utrecht-link-color: var(--todo-page-footer-link-color);
+  --utrecht-link-hover-color: var(--todo-page-footer-link-hover-color);
```

```suggestion
  --utrecht-paragraph-color: var(--todo-page-footer-color);
  --utrecht-link-color: var(--todo-page-footer-color);
  --utrecht-link-hover-color: var(--todo-page-footer-hover-color);
```

(Volgens mij zijn ze allemaal dezelfde kleur toch?)

---

✅ klopt idd, ik heb het aangepast